CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.14.2
```

Comment:
remove version number from CL; you don't know what version this will be published as.

Reply:
So this is the pattern we've used through packages.

First change post-release: bump version & -dev

So 1.2.3 becomes 1.2.4-dev

The changelog entries follow:

```
## 1.2.4
```

Then if it becomes a breaking release...

1.2.4-dev become 2.0.0-dev

and update the changelog accordingly

That's how we (try) to manage all packages.

Reply:
ok, didn't know that was a standard practice as well.  we'll try and do it that way then.

Reply:
Not to muddy the waters, but I use `## unreleased` as a placeholder.

Reply:
done